aten/src/ATen/native/cuda/jit_utils.h

```diff
@@ -228,6 +228,9 @@ template <> inline std::string typeName<at::Float8_e5m2fnuz>() {
 template <> inline std::string typeName<at::Float8_e4m3fnuz>() {
     return "at::Float8_e4m3fnuz";
 }
+template <> inline std::string typeName<at::Float8_e8m0fnu>() {
+    return "at::Float8_e8m0fnu";
```

Comment:
You already define this mapping in ScalarType ? Can the code here be made generic for any scalartype?

Reply:
tracking in https://github.com/pytorch/pytorch/issues/146647